app/models/concerns/with_reminders.rb

```diff
@@ -1,14 +1,26 @@
 module WithReminders
 
   def build_reminder
-    UserMailer.new.we_miss_you_reminder(self, cycles_since(last_submission_date))
+    last_submission_date.nil? ?
+      UserMailer.new.no_submissions_reminder(self) :
+      UserMailer.new.we_miss_you_reminder(self, cycles_since(last_submission_date))
   end
 
   def send_reminder!
     build_reminder.deliver
-    update! last_reminded: Time.now
+    update! last_reminded_date: Time.now
   end
 
+  def should_send_reminder?
+    remider_due? && (has_no_submissions? || has_no_recent_submission?)
```

Comment:
It wasn't introduced in this commit, but there's a typo in the method remider_due?

Reply:
Yup tere is a tpo

Reply:
Tipo fyxed!